### PR TITLE
Sources warning fix

### PR DESCRIPTION
**Pull request by Itee**
Previously [#10064](https://github.com/mrdoob/three.js/pull/10064), due to naming collision with upstream branch.

### CODE REVIEW

**Comment by TristanVALCKE** [comment]
Here the revert chunk for EventDispatcher. 
I will investigate why it's breaking things for an array copy...

**Comment by mrdoob** [comment]
> I will investigate why it's breaking things for an array copy...

https://github.com/mrdoob/eventdispatcher.js/pull/5

File: src/animation/AnimationAction.js [hunk 1]
```diff
@@ -473,7 +473,7 @@ AnimationAction.prototype = {
 			if ( loopCount === -1 ) {
 				// just started
 
-				this.loopCount = 0;
+				this._loopCount = 0;
```
Comment (mrdoob) [inline src/animation/AnimationAction.js]: FYI @tschw @takahirox

File: src/math/Box2.js [hunk 2]
```diff
@@ -129,27 +129,13 @@ Box2.prototype = {
 
 	containsPoint: function ( point ) {
 
-		if ( point.x < this.min.x || point.x > this.max.x ||
```
Comment (TristanVALCKE) [inline src/math/Box2.js]: It's not my favorite hobbit !!! ^^
I just try to keep code consistency between files accordingly (from yesterday, great decision to implement this tool from zz85) to the new eslint module. And your code style.

In fact, there is a lot of improvement that could be done in my view (to keep new and older programmer from a lot of pain) but, for now, i do not know what are your goal for three.js... That's my problem !

What are your plan for the next ? Es6 class and module as i can see in [#9562](https://github.com/mrdoob/three.js/issues/9562#issuecomment-261006169) ? 
I suppose, in concordance with what i read from our exchanges, that:

- you want better code (of course)
- the code still remain readable for new comer (not too more complicated or obfuscated)
- backward compatibility was keep (At least until the next major release)
- the doc was up to date (but without using auto generation from comment in code source that pollute readability).
- you would remove bower dependency
- you would keep a single configuration file for all the project

If there are other please tell me, that will avoid me to ask for PR that will be rejected for not known yet reasons !

Comment (mrdoob) [inline src/math/Box2.js]: Well... In this case, it makes the code less readable. And Closure Compiler already simplifies this when minifying to `three.min.js`.

Also, interestingly, Closure compiler produces this instead:

```js
return box.max.x < this.min.x || box.min.x > this.max.x || box.max.y < this.min.y || box.min.y > this.max.y || box.max.z < this.min.z || box.min.z > this.max.z ? false : true;
```

Which I think it's more readable.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**